workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/ComponentVisualElement.cs

```diff
@@ -12,6 +14,12 @@ public abstract class ComponentVisualElement : VisualElement
 
         protected readonly Foldout ComponentFoldout;
         protected readonly Toggle AuthoritativeToggle;
+        private static bool hideIfEmpty;
+
+        public static void ToggleHideIfEmpty(bool newValue)
+        {
+            hideIfEmpty = newValue;
+        }
```

Comment:
I'm not a fan of global and mutable state in this way, is there another way we can inject this information into the component?

Reply:
Fixed it by storing the bool in `EntityDetail`